### PR TITLE
[PARKED] position font with rendered font height and offset

### PR DESCRIPTION
**Pull request by alviney**
This has diverged from the forked repo now to suit our needs.

### CODE REVIEW

File: canvas-text-wrapper.js [hunk 1]
```diff
@@ -162,7 +164,7 @@
       checkLength(words);
       breakText(words);
 
-      textBlockHeight = lines.length * lineHeight;
+      textBlockHeight = (lines.length * opts.fontHeight) + ((lines.length - 1) * (Math.abs((opts.fontHeight * opts.lineHeight) - opts.fontHeight)));
```
Comment (dnahodil) [inline canvas-text-wrapper.js]: This is a more complicated expression now, maybe worth extracting a few intermediate variables?
e.g. giving a name to whatever `(Math.abs((opts.fontHeight * opts.lineHeight) - opts.fontHeight))` represents (is that the gap height between lines?)

**Comment by dnahodil** [comment]
Do you reckon this change is general enough to make a PR to the main repo?

**Comment by jkburges** [comment]
Can be closed?

**Comment by jkburges** [comment]
> Can be closed?

I'll assume "yes" but please re-open if needs be.

**Comment by alviney** [comment]
> I'll assume "yes" but please re-open if needs be.

Oops. Sorry I missed this. I should have closed it.

**Pull request closed without merging**